src/components/HistoryTable/HistoryTableTestSearch/HistoryTableTestSearch.tsx

```diff
@@ -8,8 +8,16 @@ import { queryString, url } from "utils";
 
 const { upsertQueryParam } = url;
 const { parseQueryString } = queryString;
+export interface HistoryTableTestSearchAnalytics {
+  failedTests: string[];
+}
+interface HistoryTableTestSearchProps {
+  sendAnalytics?: (v: HistoryTableTestSearchAnalytics) => void;
+}
 
-export const HistoryTableTestSearch = () => {
+export const HistoryTableTestSearch = ({
+  sendAnalytics,
+}: HistoryTableTestSearchProps): JSX.Element => {
```

Comment:
This is implicit.

Reply:
discussed offline

Reply:
can we use React.VFC now that we figured it out?